docs/docs/tracing/output-formats.md

```diff
@@ -92,8 +92,8 @@ output and standard error:
 
 2. Error file
 
-    Redirect errors to your log files if needed:
+    Redirect logs to a file if needed:
 
     ```text
-    $ sudo TRACEE_BPF_FILE=do-not-exist ./dist/tracee-ebpf --output json --trace comm=bash --trace follow --trace event=openat --output out-file:/tmp/tracee.log --output err-file:/tmp/tracee.err
+    $ sudo TRACEE_BPF_FILE=do-not-exist ./dist/tracee-ebpf --output json --trace comm=bash --trace follow --trace event=openat --output out-file:/tmp/tracee.log --output log-file:/tmp/tracee.log
```

Comment:
tracee.log is used in this example both for out-file and log-file. Better change one of them to something else

Reply:
`tracee.events` and `tracee.log` respectively?

Reply:
> `tracee.events` and `tracee.log` respectively?

Can be. The main point is to have different files